configs/razor_subnets/yolov5_s_spos_shufflenetv2_syncbn_8xb16-300e_coco.py

```diff
@@ -0,0 +1,40 @@
+_base_ = [
+    'mmrazor::_base_/nas_backbones/spos_shufflenet_supernet.py',
+    'mmyolo::yolov5/yolov5_s-v61_syncbn_fast_8xb16-300e_coco.py'
+]
+
+custom_imports = dict(imports=['mmrazor.models'], allow_failed_imports=False)
+checkpoint_file = 'https://download.openmmlab.com/mmrazor/v0.1/nas/spos/spos_shufflenetv2_subnet_8xb128_in1k/spos_shufflenetv2_subnet_8xb128_in1k_flops_0.33M_acc_73.87_20211222-1f0a0b4d.pth'  # noqa
```

Comment:
For MMRazor dev-1.x branch, the URL should be https://github.com/open-mmlab/mmrazor/blob/dev-1.x/configs/nas/mmcls/spos/metafile.yml#L20

Our SPOS readme also needs to update the checkpoint URL